Form.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+	render() {
+		return (
+			<div><div data-reactid=".1" data-react-checksum="-878755164"><button type="button" class="btn" data-reactid=".1.$0"><span data-reactid=".1.$0.0"> </span><span data-reactid=".1.$0.1">Button</span><span data-reactid=".1.$0.2"> </span></button></div></div>
+		);
+	}
+}
```

Comment:
There are a lot of issues here.  Why is it using React.Component instead of a stateless component?  Why is it specifying data-react* attributes?